src/makeInfinityAdder.js

```diff
@@ -7,6 +7,22 @@
 
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  const add = (num) => {
+    if (!num) {
```

Comment:
This will work incorrectly if we call `add(0)`.
Need to use different check for `num` presence.